test/album.spec.js

```diff
@@ -0,0 +1,68 @@
+const request = require('supertest');
+const app = require('../app');
+const userModel = require('../app/models').user;
+const userAlbumModel = require('../app/models').user_album;
+const authenticationService = require('../app/services/authentication');
+const methods = require('../app/services/album');
+methods.getAlbumById = jest.fn(id => Promise.resolve({ title: 'The title', id }));
```

Comment:
Rename this to albumService or something like that, not needed for training but normally mocks are setted for each test, instead of general successful